python/ccxt/async_support/base/ws/fast_client.py

```diff
@@ -35,7 +35,7 @@ def feed_data(message, size):
             self.stack.append(message)
 
         def feed_eof():
-            if self._close_code == 1000:  # OK close
+            if self.connection._close_code == 1000:  # OK close
                 self.on_close(1000)
             else:
                 self.on_error(1006)  # ABNORMAL_CLOSURE
```

Comment:
I think we also need to change this to `on_error(NetworkError(1006))` to make it raise an exception